test/cri-containerd/container_test.go

```diff
@@ -874,3 +874,73 @@ func Test_RunContainer_ExecUser_Root_LCOW(t *testing.T) {
 		t.Fatalf("expected user for exec to be 'root', got %q", string(r.Stdout))
 	}
 }
+
+// creates two sandboxes and starts two containers within them. The first sandbox then starts a NFS server and
```

Comment:
"The first **container** then"?

Reply:
also, I only see one sandbox and one container, is this comment outdated?

Reply:
Yeah, original plan was to have two containers in different sandboxes, but running NFS server requires additional modules in the kernel that we don't have. Fixed the comment now.